store/localstore/mvcc_bench_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2015 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package localstore
+
+import (
+	"fmt"
+	"math/rand"
+
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/store/localstore/goleveldb"
+	"github.com/pingcap/tidb/util/codec"
+)
+
+var _ = Suite(&testMvccBenchSuite{})
+
+type testMvccBenchSuite struct {
+	s *dbStore
+}
+
+// Open store for benchmark
+func createStore(testName string) kv.Storage {
+	path := fmt.Sprintf("db-%s", testName)
+	d := Driver{
+		goleveldb.Driver{},
+	}
+	store, err := d.Open(path)
+	if err != nil {
+		panic(err)
+	}
+	return store
+}
+
+var randLetters = []byte("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+func randBytes(r *rand.Rand, size int) []byte {
```

Comment:
if we only want randBytes, https://golang.org/pkg/crypto/rand/#Read is simpler. If we do want rand letter, probably we should rename randBytes to randLetters.